test/module-global.js

```diff
@@ -0,0 +1 @@
+window.module = {};
```

Comment:
No, it provides `module` so pixelmatch works without bundling. It has `module.eports = pixelmatch` in it.